src/calf/orfanidis_eq.h

```diff
@@ -748,7 +748,7 @@ class EllipticTypeBPFilter : public BPFilter {
 			std::vector<eq_double_t> v = landen(k, tol);
 
 			std::transform(v.begin(), v.end(), v.begin(),
-			    bind2nd(std::plus<eq_double_t>(), 1.0));
+			    bind(std::plus<eq_double_t>(), 1.0, std::placeholders::_1));
```

Comment:
Just for me to understand: You switched around 1st and second argument here, but it does not matter because `double` addition is commutative?

Reply:
I read that `std::placeholders` is zero-indexed, so `std::placeholders::_1` is the second position of a function

Reply:
I looked at https://stackoverflow.com/questions/32739018/a-replacement-for-stdbind2nd, where they replaced `std::bind2nd(std::greater<int>(), bar)` by `std::bind(std::greater<int>(), _1, bar)`, so it looks like either the SO answer or this PR are switching the arguments? (obviously, nit-picking, I just want to make sure I understand it all)

Reply:
> I looked at https://stackoverflow.com/questions/32739018/a-replacement-for-stdbind2nd, where they replaced `std::bind2nd(std::greater<int>(), bar)` by `std::bind(std::greater<int>(), _1, bar)`, so it looks like either the SO answer or this PR are switching the arguments? (obviously, nit-picking, I just want to make sure I understand it all)

Oh, yep. Looks like I switched them around. Will fix.